todo_app/test_view_model.py

```diff
@@ -0,0 +1,34 @@
+from todo_app.data.view_model import ViewModel
+from todo_app.Item import Item
+
+def test_view_model_done_items():
+    # Arrange
+    items = [
+        Item("1","New ToDo", "To Do"),
+        Item("2","Completed task", "Done")
+    ]
+    view_model = ViewModel(items)
+
+    # Act
+    returned_items = view_model.done_items
+
+    # Assert
+    assert len(returned_items) == 1
+    single_item = returned_items[0]
+    assert single_item.status == "Done"
+
+def test_view_model_to_do_items():
+    # Arrange
+    items = [
+        Item("1","New ToDo", "To Do"),
+        Item("2","Completed task", "Done")
+    ]
+    view_model = ViewModel(items)
```

Comment:
Minor, but since the setup here is identical to the previous unit test, it might encourage us to pull out a fixture to share that, e.g. this might look like:
```
import pytest

@pytest.fixture
def view_model():
    items = [
        Item("1","New ToDo", "To Do"),
        Item("2","Completed task", "Done")
    ]
    return ViewModel(items)

def test_view_model_done_items(view_model):   

    # Act
    returned_items = view_model.done_items```
    <.....>

def test_view_model_to_do_items(view_model):
    # Act
    returned_items = view_model.to_do_items
    <.......>